src/modules/daemonconfiguration/src/daemonconfiguration.rs

```diff
@@ -82,19 +121,34 @@ impl SystemctlInfo for Systemctl {
             if service["service"].ends_with("@") {
                 continue;
             }
-            let daemon = Self::create_daemon(&service["service"], &service["status"])?;
-            services.push(daemon);
+            let daemon = self.create_daemon(&service["service"], &service["status"])?;
+            // Only report enabled daemons with State not being Other or dead
+            if (daemon.state != State::Other)
+                && (daemon.state != State::Dead)
+                && (daemon.auto_start_status == AutoStartStatus::Enabled)
+            {
+                services.push(daemon);
+            }
         }
         Ok(services)
     }
 
-    fn create_daemon(name: &str, auto_start_status: &str) -> Result<Daemon, MmiError> {
-        let substate = Self::show_substate_property(name)?;
+    fn list_unit_files(&self) -> Result<String> {
```

Comment:
>{ [](http://example.com/codeflow?start=48&length=1)

Lines 129 - 131 do it differently. If Rust supports that format, use it everywhere for consistency:

statement
{
    ....
}

rather than:

statement {
    ....
}

Reply:
It's actually consistent, just different for single line statements vs multi-line statments:
statement {

}

if statement &&
  statement 
{

}

But official rust convention seems to be ^^. I used the rust formatter which follows the official guidelines: https://rust-lang.github.io/rfcs/2436-style-guide.html